src/components/Modal/Modal.styled.jsx

```diff
@@ -41,7 +41,7 @@ export const InfoContainer = styled.div`
     font-size: 24px;
     color: #005D9D;
     margin-bottom: 10px;
-    margin-top: 25px;
+    margin-top: 5px;
```

Comment:
Remember you're changing global modal styles, you can set just the delivery reached modal instead